src/pages/Users/Students/Students.tsx

```diff
@@ -463,7 +476,27 @@ export const Student: React.FC<{
   });
 
   function handleReset() {
-    setValues({});
+    setValues({
```

Comment:
Create a separate static obj for this, use while initializing as well while resetting.